app/src/features/projects/ProjectsRouter.tsx

```diff
@@ -27,14 +28,18 @@ const ProjectsRouter: React.FC = () => {
     <Switch>
       {/* Project List Routes */}
       <RouteWithTitle exact path="/admin/projects" title={getTitle('Projects')}>
-        <ProjectsListPage />
+        <DialogContextProvider>
```

Comment:
Any particular reason for wrapping all our pages with `<DialogContextProvider`, rather than just wrapping the entire router? Perhaps to support multiple active dialogs at once?

Reply:
TLDR: performance reasons related to pages where more than 1 large component is being rendered (ex: the observations table + sampling site list).